common_features.mk

```diff
@@ -240,6 +246,8 @@ ifeq ($(strip $(LEADER_ENABLE)), yes)
   OPT_DEFS += -DLEADER_ENABLE
 endif
 
+include $(DRIVER_PATH)/qwiic/qwiic.mk
```

Comment:
I'm not exactly fond of having those rules in a different file.  There should be no issue with including the contents of the file here.

Reply:
i think jack did it this way to make it easier to add new qwiic devices, keeps the commits to the same folder

Reply:
@jackhumbert ?